pyanaconda/modules/payloads/payloads.py

```diff
@@ -114,7 +114,14 @@ def generate_kickstart(self):
             log.debug("Generating kickstart... (skip)")
             return ""
 
-        return super().generate_kickstart()
+        log.debug("Generating kickstart...")
+        handler = self.get_kickstart_handler()
+        self.setup_kickstart(handler)
+
+        # FIXME: This is a temporary workaround for packages.
```

Comment:
I think some information about why and till when the workaround is needed would be useful.

Reply:
Added another comment.